test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -0,0 +1,18 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import sys
+import onnxruntime  # noqa
+
+
+class TestONNXRuntime(unittest.TestCase):
+
+    def test_onnxruntime_installed(self):
+        self.assertTrue('onnxruntime' in sys.modules)
```

Comment:
I assume this will be removed once meaningful tests are added in the future?

Reply:
absolutely!